server/src/test/java/io/crate/execution/engine/collect/DocValuesGroupByOptimizedIteratorTest.java

```diff
@@ -52,6 +52,8 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import com.google.common.base.Predicates;
```

Comment:
Where's the guava coming from? We don't have this declared as dependency in server and I don't think we want to reintroduce it?

Reply:
Good point. It's a dependency of `com.auth0:jwks-rsa`, but I'll remove the usage.